backend/src/shared/interface/modelInterface.ts

```diff
@@ -0,0 +1,28 @@
+import { Document } from "mongoose";
+
+export interface ProfileInterface extends Document {
```

Comment:
I think we are missing 'college', 'classes', 'hobby', and 'contact' (based off figma)